STM32L4/luos_hal.c

```diff
@@ -745,3 +749,217 @@ void LuosHAL_FlashReadLuosMemoryInfo(uint32_t addr, uint16_t size, uint8_t *data
 {
     memcpy(data, (void *)(addr), size);
 }
+
+#ifdef BOOTLOADER_CONFIG
```

Comment:
I prefere to have all the function at the end of the file with only one #ifdef

Reply:
OK, I made these changes.